content/pt-br/docs/reference/kubectl/cheatsheet.md

```diff
@@ -439,7 +443,7 @@ kubectl get pods -A -o=custom-columns='DATA:metadata.*'
 
 Mais exemplos no kubectl [documentação de referência](/docs/reference/kubectl/#custom-columns).
```

Comment:
Sounds better to me "Mais exemplos na documentação de referência do kubectl". What do you think?